tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -319,4 +319,83 @@ public function testListFloatTypeColumns()
         $this->assertFalse($columns['col']->getUnsigned());
         $this->assertTrue($columns['col_unsigned']->getUnsigned());
     }
+
+    /**
+     * Test default null values (especially for mariadb 10.2.7+)
+     * @link https://github.com/doctrine/dbal/issues/2817
+     */
+    public function testNullColumnDefaultWithTable()
+    {
+        $table = new Table("text_null_default_value");
+        $table->addColumn('txt_def_null', 'string', ['notnull' => false, 'default' => null]);
+        $table->addColumn('txt_def_null_as_string', 'string', ['notnull' => false, 'default' => 'NULL']);
+        $table->addColumn('txt_def_null_as_quoted_string', 'string', ['notnull' => false, 'default' => "NULL"]);
```

Comment:
How is this different from the field above?

Reply:
Fixed, thanks